src/System.Text.Json/tests/Serialization/TestData.cs

```diff
@@ -12,6 +12,7 @@ public static IEnumerable<object[]> ReadSuccessCases
         {
             get
             {
+                yield return new object[] { typeof(SimpleTestStruct), SimpleTestStruct.s_data };
```

Comment:
If the goal of this change is to provide struct support in the serializer, we should consider adding a lot more test cases, including nested types (struct within reference, reference within struct, etc.).

Reply:
Haven't done that because `struct`s have no difference from primitive value types and `decimal` which isn't primitive. The idea of that test was to check that setting a value on a struct instance sets the value on the original struct on the heap and not on its stack allocated copy. But if it's required, I can write more tests. Should I do that and should I duplicate tests of `SimpleTestClass`?

Reply:
We need to at least verify multiple properties are applied from json.

Reply:
d